modules/_labs/combobox/react/lib/AutocompleteList.tsx

```diff
@@ -0,0 +1,133 @@
+import React, {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+import {spacing} from '@workday/canvas-kit-react-core';
+import {accessibleHide} from '@workday/canvas-kit-react-common';
+import {MenuItemProps} from '@workday/canvas-kit-labs-react-menu';
+import {ComboBoxMenuItemGroup, getOptionId, listBoxIdPart, getTextFromElement} from './Combobox';
+import uuid from 'uuid/v4';
+
+const Autocomplete = styled('ul')({
+  padding: 0,
+  margin: `${spacing.xxs} 0`,
+});
+
+const AccessibleHide = styled('span')({
+  ...accessibleHide,
+});
```

Comment:
will this work?
```
const AccessibleHide = styled('span')(accessibleHide);
```

Reply:
just my personal opinion,I dont think its difficult to add three dots and comma when we need it. 
What if we will never need it? we will be just carrying this extra line and a unnecessary copy of the object, for indefinite time.

Reply:
I would argue that lines are free, and in this case the spread cost is a bit of a micro optimization. 
I do think there is probably some places this module can benefit from performance improvements but not sure if this would move the needle.